torchrl/objectives/costs/ppo.py

```diff
@@ -81,6 +81,17 @@ def __init__(
         self.loss_critic_type = loss_critic_type
         self.advantage_module = advantage_module
 
+    def make_functional_with_buffers(self):
+        self.actor, (
+            actor_params,
+            actor_buffers,
+        ) = self.actor.make_functional_with_buffers()
+        self.critic, (
+            critic_params,
+            critic_buffers,
+        ) = self.critic.make_functional_with_buffers()
+        return (actor_params, actor_buffers, critic_params, critic_buffers)
```

Comment:
maybe we can return a class here - that wraps over this tuple (though I am not sure how much extra value that adds).

Reply:
see updated PR :)